core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -350,4 +354,16 @@
     private RegistryManagementConstants() {
         // prevent instantiation
     }
+
+    /**
+     * Checks if a given secret is of type {@value #SECRETS_TYPE_HASHED_PASSWORD}.
+     *
+     * @param secret The secret to check.
+     * @return {@code true} if the secret has a string property with name {@value #FIELD_TYPE}
+     *         and value {@value #SECRETS_TYPE_HASHED_PASSWORD}.
+     */
+    public static boolean isHashedPasswordSecret(final JsonObject secret) {
```

Comment:
Missing info regarding `NullPointerException` in _JavaDoc_ if the secret is _null_.

Reply:
This method is not used anyway, I'll remove it ...